CHANGELOG.md

```diff
@@ -38,14 +43,6 @@ This release extends the planned support of the modules to OneView REST API vers
 #### Bug fixes & Enhancements
 - [#489](https://github.com/HewlettPackard/oneview-ansible/issues/489) Server Profile - create operation fails with an error -Value specified for enclosure is not valid or not supported
 
-
-## v5.6.0(unreleased)
-This release extends the planned support of the modules to OneView REST API version 800(OneView v4.1), 1000(OneView v4.2) and 1200(OneView v5.0) and 1600(Oneview v5.2).
-
-#### Modules supported in this release
-- oneview_server_profile_template
```

Comment:
Don't remove these resources from CHANGELOG.

Reply:
modified the changelog